ktor-http/ktor-http-cio/src/io/ktor/http/cio/internals/WeakTimeoutQueue.kt

```diff
@@ -74,34 +74,29 @@ class WeakTimeoutQueue(
     suspend fun <T> withTimeout(block: suspend CoroutineScope.() -> T): T {
```

Comment:
KDoc to with timeout: "Execute [block] and cancel if doesn't complete it time." 
it time -> in time